openncp-client/manifests/openncp-client-service.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: openncp-client-service
+  namespace: openncp
+spec:
+  type: NodePort
+  ports:
+  - name: http
+    port: 7080
+    targetPort: 7080
+    nodePort: 30002
```

Comment:
Typically we do not hard code the nodePort as it may be already assigned to another service.  Kubernetes manages the nodePort number dynamically, and thus doesn't need to be assigned.

Reply:
Commented the nodePort on each service! It is up to admins to define ports. Added also a comment for this.